src/ui/display.c

```diff
@@ -252,7 +252,7 @@ bool skip_change_outputs() {
         return false;
     }
     // confirmed outputs holds the true current output index
-    uint8_t change_indices[1 + TX_MAX_TOKENS];
+    uint8_t change_indices[1 + TX_MAX_TOKENS] = {0};
```

Comment:
Same `core.UndefinedBinaryOperatorResult` error, so zero initializing the array solves the issue.